code-challenges.js

```diff
@@ -10,7 +10,42 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const sent = (obj) => {
 
+  let length = obj.length
+
+  let arrNames = obj.map(value => value.name)
+  // let arrCap = []
+
+  // for (let i = 0; i < arrNames.length; i++) {
+  //   arrCap.push(arrNames[i].split(" "))
+  // }
+
+  let arrCap = arrNames.map(value => value.split(" "))
+
+  let name1 = arrCap.map(value => value[0].charAt(0).toUpperCase() + value[0].slice(1))
+  let name2 = arrCap.map(value => value[1].charAt(0).toUpperCase() + value[1].slice(1))
+
+  let names = name1.concat(name2)
+
+  let occ = obj.map(value => value.occupation)
+
+  let finalArray = []
+
+  for (let i = 0; i < length; i++) {
+    finalArray.push(`"${name1[i]} ${name2[i]} is a ${occ[i]}." `)
+  }
+
+  return finalArray.join("")
+
```

Comment:
Hey, Sami! This looks awesome! I think my only feedback would be to pseudo code your work in order to let others understand the way you tackle coding challenges.